playbook/app/pb_kits/playbook/_reset.scss

```diff
@@ -40,3 +40,7 @@ a {
   text-decoration: none;
   color: $primary;
 }
+
+.sandbox-preview-iframe body {
+  background-color: black !important;
+}
```

Comment:
This is used for any client. I suggest moving to site styles or someplace in website.

https://github.com/powerhome/playbook/blob/master/playbook-website/app/javascript/site_styles/_site-style.scss

Reply:
The background of the reset.css is actually bleeding INTO the iframe of the component. When I set it in the website, as you suggested, it gets overwritten by the lines ABOVE this one, where i added the background.

Reply:
We either have to make the reset a little more flexible or we have to bring in another separate file into every sandpack example that overrides that greedy code.

Reply:
@jasoncypret